crates/bevy_ecs/src/component.rs

```diff
@@ -346,11 +364,34 @@ impl Components {
         self.components.get_unchecked(id.0)
     }
 
+    /// Type-erased equivalent of [`Components::component_id`].
     #[inline]
     pub fn get_id(&self, type_id: TypeId) -> Option<ComponentId> {
         self.indices.get(&type_id).map(|index| ComponentId(*index))
     }
 
+    /// Retrieves the [`ComponentId`] of the given [`Component`] type in
+    /// this [`Components`] instance.
+    ///
+    /// Returns [`None`] if the [`Component`] type has not
+    /// yet been initialized using [`Components::init_component`].
+    /// ```rust
```

Comment:
```suggestion
    /// yet been initialized using [`Components::init_component`].
    ///
    /// ```rust
```

Adds an empty line between the paragraph and the code block (only in the code, not in the generated docs). It will improve readability.